server/src/main/java/chat/engine/Connect.java

```diff
@@ -108,13 +114,27 @@ private void closeConnection() throws IOException {
         reader.close();
         writer.close();
         clientsOnline.remove(name);
+    }
+
+    private void sendMessage(String message, String name) throws IOException {
 
+        for (Map.Entry<String, Socket> entry : clientsOnline.entrySet()) {
+            if (entry.getValue() != socket) {
+                writer = getWriter(entry.getValue());
+                writer.println(name + ": " + message);
+            }
+        }
     }
 
-    private void sendMessage(String message) throws IOException {
+    private boolean privateMessage(String message, String name) throws IOException {
```

Comment:
you used return type boolean, but did not use this value, you should make this method 'void', but It seems to be better if you write to user who tried to send PM something like 'this user is offline' in case, when your method return false